### PR TITLE
#738 ZK & BK service info display is incomplete

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #738 )
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/Services/ZookeeperListPage/ZookeeperListPage.js [hunk 1]
```diff
@@ -15,20 +15,24 @@
...
+import { isEmpty, map, get } from 'lodash';
 
 import * as zookeeperApi from 'api/zookeeperApi';
 import { Box } from 'common/Layout';
 import { H2 } from 'common/Headings';
 import { TableLoader } from 'common/Loader';
-import { FormGroup, Input, Label } from 'common/Form';
+import { FormGroup } from 'common/Form';
+
+import * as s from '../styles';
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/Services/ZookeeperListPage/ZookeeperListPage.js]: It looks like we're only using a few styles from the upper-level component's `styles.js`. Can we either make a `styles.js` here in the `/ZookeeperListPage` directory or just reuse the styles form `/BrokerListPage`?

Comment (wu87988622) [reply ohara-manager/client/src/components/pages/Services/ZookeeperListPage/ZookeeperListPage.js]: ok I know how to do it.

**Comment by konekoya** [comment]
@wu87988622 could you fix the merge conflict and push again?

**Comment by konekoya** [comment]
Awesome! Please merge this PR once the QA passes

**Pull request merged**